samples/sensor/lidar_lite_v4/src/main.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025 Alex Aylward
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/kernel.h>
+
+int main(void)
```

Comment:
Please drop the code sample. There is alreadyhttps://docs.zephyrproject.org/latest/samples/sensor/distance_polling/README.html#distance_polling that's effectively the same